lib/plugins/plugins.js

```diff
@@ -350,6 +350,13 @@ const getAppViews = additionalViews => expandToIncludeShadowNunjucks(self.getFil
   .reverse()
   .concat(additionalViews || [])
 
+function getConstants () {
+  return Object.fromEntries(
+    getList('constants')
+      .map(({ item: { name, value } }) => [name, value])
+  )
+}
+
```

Comment:
This suggestion came from @nataliecarey's [`frontend-v5-compatibility` branch](https://github.com/alphagov/govuk-prototype-kit/commit/cb3c4c5ea2a6624a8cdf4a1f5305fffb361d2f61#diff-1a28c09ab9cc522f4e25b2162711b3064b269a5fccf5289b9f1e80dd7b3492cdR370-R378)

Perhaps there's a better way since the `meta` field came out?